backend/src/handler.ts

```diff
@@ -68,45 +68,49 @@ server.use((req: any, res: any, next: any) => {
 
 const sls = serverlessHttp(server);
 const defaultConfig = new AppConfig();
-const config = JSON.parse(fs.readFileSync("./config/appconfig.json", "UTF-8"));
+const config = JSON.parse(fs.readFileSync("./config/appconfig.json", "utf8"));
 const appConfig = AppConfig.merge(defaultConfig, config);
 const environment = new CloudEnvironment();
 const swagger = new Swagger(
   server,
   new SwaggerConfig(appConfig.readOnly, appConfig.enableApiKeyAuth),
-  environment.basePath,
-  "./package.json"
+  environment.basePath
 );
+
 let core: CoreApp | undefined;
-console.log("environment: " + process.env.NODE_ENV);
-console.log("isoffline: " + process.env.IS_OFFLINE);
-if (process.env.IS_OFFLINE) {
+const init = async () => {
+
+  console.log("environment: " + process.env.NODE_ENV ? process.env.NODE_ENV : 'dev');
+  console.log("isoffline: " + process.env.IS_OFFLINE);
+
+  if (process.env.IS_OFFLINE) {
   core = new CoreApp(
-    appConfig,
-    server,
-    new FileStorageAdapter("db.json"),
-    swagger,
-    environment
-  );
-} else {
+      appConfig,
+      server,
+      new FileStorageAdapter('db.json'),
+      swagger,
+      true
```

Comment:
that`s interesting why this can be true - I need to test and build this repo again to figure out why ...